### ISSUE TITLE
Request for features and bug fixes

### ISSUE THREAD

**Issue by f4rh4d-4hmed** [rebelonion/Dantotsu#500]
I'll be requesting for a feature and letting know of some bugs I have found while using this app. 
Bugs/Errors to fix : 
1. While watching video, sometimes while trying to manually skip OP by using seek, I mistakenly clicks the server choice button. Than the server starts loading. But if I try ignore it and watch the video, the video won't play anymore until I select a server manually. If I select same server as before, it just starts loading/downloading video over again.
2. The "ANIME" section lacks a "Top Airing" category. While it has sections like "Top Rated" and "Trending Movie," there's no dedicated section for currently airing shows ranked by rating.

My request would be to fix that bug and add that feature.

**Comment by grayankit** [comment]
I will see what can be done

**Comment by itsmechinmoy** [comment]
> 2. The "ANIME" section lacks a "Top Airing" category. While it has sections like "Top Rated" and "Trending Movie," there's no dedicated section for currently airing shows ranked by rating.

Does anilist have it?

**Comment by f4rh4d-4hmed** [comment]
> > 2. The "ANIME" section lacks a "Top Airing" category. While it has sections like "Top Rated" and "Trending Movie," there's no dedicated section for currently airing shows ranked by rating.
> 
> Does anilist have it?

Nope, Anilist API does not have it, but with filtering this can be achieved, I think

**Comment by rebelonion** [comment]
> The "ANIME" section lacks a "Top Airing" category. While it has sections like "Top Rated" and "Trending Movie," there's no dedicated section for currently airing shows ranked by rating.


I don't think this is worthwhile tbh
